### PR TITLE
to control whether a `socket` can be used

### PR DESCRIPTION
**Pull request by NicoChenyt**
I don't want to run all `socket` on every machine, so i need to control whether a `socket` can be used by config.

### CODE REVIEW

File: src/Illuminate/LaravelSCommand.php [hunk 1]
```diff
@@ -153,6 +153,10 @@ protected function showProtocols()
         ];
         $sockets = isset($config['server']['sockets']) ? $config['server']['sockets'] : [];
         foreach ($sockets as $key => $socket) {
+            if (isset($socket['enable']) && !$socket['enable']) {
```
Comment (hhxsv5) [inline src/Illuminate/LaravelSCommand.php]: This is just to show the list of sockets, you need to add the same judgment logic [here](https://github.com/hhxsv5/laravel-s/blob/master/src/Swoole/Server.php#L127), and then explain it on the documents.

Comment (hhxsv5) [reply src/Illuminate/LaravelSCommand.php]: Look forward to your commits.

Comment (NicoChenyt) [reply src/Illuminate/LaravelSCommand.php]: Its my mistake. I resubmitted the code

**Comment by hhxsv5** [comment]
Thanks, merged.

**Pull request merged**